shop-be/import-service/src/services/stream.ts

```diff
@@ -0,0 +1,51 @@
+import { Product } from "src/types/product"
+import { Transform } from "stream"
+import { productServices } from "./db"
+import { sqsServices } from "./sqs"
+import { SQSClient } from "@aws-sdk/client-sqs";
+import { config } from '../../config'
+
+const { REGION } = config;
+
+export const stream = {
+    outputToQueue: () => {
+        const client = new SQSClient({ region: REGION });
+
+        return new Transform({
```

Comment:
Also there is https://github.com/rvagg/through2 library, that can be used like a wrapper above transform stream - code would be more concise